index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <title>Mnemba Chambuya</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="flex-nav">
+        <div class="logo"><h3>Mnemba Chambuya</h3></div>
```

Comment:
- [ ]  It's required that we enclose this Logo in an anchor `<a>` tag. Please consider refactoring it. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.